package.json

```diff
@@ -3,12 +3,11 @@
 	"version": "0.0.1",
 	"description": "This is a boilerplate of an Apify actor.",
 	"dependencies": {
-		"apify": "^0.15.2",
-		"moment": "^2.24.0",
-		"request-promise": "^4.2.4",
-		"underscore": "^1.9.1"
+		"apify": "^2.0.1",
+		"got-scraping": "^2.1.2",
```

Comment:
I guess you could use `requestAsBrowser` instead of requiring `got-scraping` directly

Reply:
Yeah, sorry, that's true, somehow not using the fingerprint generator confused me that I should use explicitly the got-scraping